cargo-nextest/src/dispatch.rs

```diff
@@ -132,6 +154,11 @@ enum Command {
 
         #[clap(flatten)]
         reporter_opts: TestReporterOpts,
+
+        /// The list of Rust test binaries produced by `list-bins` in json.
+        /// Allow for splitting the tests build step from the tests run step.
+        #[clap(long)]
+        test_list: Option<Utf8PathBuf>,
```

Comment:
Which heading does this show up in? I think it should just be in `OPTIONS`, I don't see a good place to put it in otherwise.

```suggestion
        bin_list: Option<Utf8PathBuf>,
```

Reply:
changed see new comment